inspirehep/dojson/experiments/fields/bd1xx.py

```diff
@@ -118,31 +136,50 @@ def collaboration(self, key, value):
 @utils.for_each_value
 def related_experiments(self, key, value):
     """Related experiments."""
-    try:
-        recid = int(value.get('0'))
-    except (TypeError, ValueError):
-        recid = None
+    def _get_record(zero_values):
+        zero_value = force_single_element(zero_values)
+        if zero_value:
+            try:
+                recid = int(zero_value)
+                return get_record_ref(recid, 'experiments')
+            except (TypeError, ValueError):
+                return None
+
+    def _classify_relation_type(w_values):
+        w_value = force_single_element(w_values)
+        if w_value:
+            if w_value == 'a':
+                return 'predecessor'
+            elif w_value == 'b':
+                return 'successor'
+            else:
+                return ''
+
+    record = _get_record(value.get('0'))
+
     return {
-        'name': value.get('a'),
-        'record': get_record_ref(recid, 'experiments')
+        'name': force_single_element(value.get('a')),
```

Comment:
Only 3: https://github.com/inspirehep/inspire-next/blob/96160928a59378acf6e1854030771a5693e304c5/inspirehep/dojson/current_marcxml_usage/experiments-repeatable.txt#L24

And it's probably not entirely correct, because it's discarding the extra data.